common/src/View/ViewPreferencePane.cpp

```diff
@@ -79,6 +81,18 @@ namespace TrenchBroom {
             setLayout(layout);
         }
 
+        namespace {
```

Comment:
Why the anonymous namespace if you're using static methods?

Reply:
Because we don't have a coding standard I guess, I did some research and this was suggested as a good style on stackoverflow.

Of the 3 possible styles the only one I don't want to use is anonymous-namespace only, because the important information that the function is static is possibly off-screen.

Reply:
What's the benefit of having both static and an anonymous namespace?

Reply:
I don't know, I just picked it because your code uses anonymous namespaces. I don't care what we do other than the point I made above. Do you prefer just static, then?

Reply:
Oh, ok, I see. I thought we had agreed that we'd be only using static from now on. I have used an anonymous namespace in some places because that's what we use at work, but that was before we had that discussion. I'm okay with using either, but not both, and since we already agree on static, I'd suggest we use that consistenly from now on.

I'll create an issue about documenting this in the style guide and removing the 7 anonymous namespaces we currently have.

Reply:
Sorry, I forgot we agreed on only `static`. Sounds good!

Reply:
No problem, I should have put it into the style guide right away.